### DISCUSSION TITLE
Character conversions

### DISCUSSION THREAD

**Discussion by petermr** [petermr/semanticClimate#37] [General]
Many entries contain characters with unknown encodings and have to be deduced and edited.
Examples are:
* guillemets
* em-dash
* degree symbol 
* chemical formulae - reaction arrow
Please add a Comment for each of these separately.

**Comment by flower1430** [comment]
- notes
- references
- superscripts
- single quote

**Comment by petermr** [comment]
character encodings

```
The point at which an actorâs objectives (or system needs) cannot be secured from intolerable risks through adaptive actions.
â¢ Hard adaptation limit â No adaptive actions are possible to avoid intolerable risks.
â¢ Soft adaptation limit â
```
# em-dash and right-smart-single-quote 
These appear to map to `â` (ahat euro tab)
single the em-dash is probably preceded by space it may be  possible to distinguish them

# bullet
Appears to be `â¢` (ahat euro cent)

**Comment by flower1430** [comment]
These are some symbols in the text
- 2015â2030  ( 2015–2030 )
- countriesâ ( countries’ )
- 3.625 Ã 1000 m2 (3.625 × 1000 m2  )
- 1000 kg mâ3, and divided  (of 1000 kg m–3, and divided)

**Comment by gayatritaneja** [comment]
CHEMICAL REACTIONS
Calcification.html
<img width="641" alt="image" src="https://github.com/petermr/semanticClimate/assets/63981052/9c9c997a-4e27-4369-aa46-87f373f04527">
<img width="683" alt="image" src="https://github.com/petermr/semanticClimate/assets/63981052/804861f5-3ef7-49a3-a748-d290310b4774">

**Reply by gayatritaneja** [reply]
REFERENCES
Benthic.html
<img width="683" alt="image" src="https://github.com/petermr/semanticClimate/assets/63981052/442cb4a4-bcc7-475d-820f-c2e17ca39b63">

**Reply by gayatritaneja** [reply]
PARENTHESIS IN TITLE
Canopy_temperature.html
Carbon_dioxide__CO2__fertilisation.html
<img width="878" alt="image" src="https://github.com/petermr/semanticClimate/assets/63981052/2400bdba-2cf1-41d2-bdcf-b7420c928006">
<img width="957" alt="image" src="https://github.com/petermr/semanticClimate/assets/63981052/3b690795-bdf7-400a-9d43-b4a4bc1133a7">

**Reply by gayatritaneja** [reply]
2 DIFFERENT DEFINITIONS & NOTES IN DESCRIPTION
Carbon_budget.html
<img width="949" alt="image" src="https://github.com/petermr/semanticClimate/assets/63981052/502a7ca0-5ad5-4eef-985a-bff604fdb1bf">

**Reply by gayatritaneja** [reply]
IMAGE REFERENCE IN DESCRIPTION
Cold_days_cold_nights.html
<img width="957" alt="image" src="https://github.com/petermr/semanticClimate/assets/63981052/bced1287-d470-43b7-ba6b-d6afe4494a4f">

**Reply by gayatritaneja** [reply]
Only gives the full forms, unclear definitions
CMIP3__CMIP5_and_CMIP6.html
<img width="758" alt="image" src="https://github.com/petermr/semanticClimate/assets/63981052/27d94dfa-32ea-46ae-9a32-45b24fd2a780">
